packages/malloy/src/dialect/dialect.ts

```diff
@@ -172,6 +177,18 @@ export abstract class Dialect {
   // Like characters are escaped with ESCAPE clause
   likeEscape = true;
 
+  // Added field supportsLimit. TSQL Doesn't support limit and limit is not part of the SQL Standard
+  // TODO (vitor): Discuss with malloy
+  supportsLimit = true;
+
+  // Added field supportsRegexp. TSQL Doesn't support regexp. Idk how to use this with = false yet (tsql case). Malloy seems to often go for regexp
+  // TODO (vitor): Discuss with malloy
+  supportsRegexp = true;
```

Comment:
There are regular expressions in Malloy expressions, but no databases "support" regular expressions as a data type. A regular expression is just a string which has special quoting rules and which therefore acts differently with the `~` operator. I don't think you need to mark them "not supported". If you can implement `sqlRegexpMatch`, which I think you can, then that is all the "support" needed.